v2.2/managed-frequently-asked-questions.md

```diff
@@ -0,0 +1,82 @@
+
+### What machines do you run your cluster on?
+
+Our machines run with 4 CPUs, 8 CPUs, or 16 CPUs per node.
+
+## Cluster maintenance
+
+### How do I change the configurations on my cluster?
+
+Today, you can reach out to [Support]([Support](https://support.cockroachlabs.com/hc/en-us) for most cluster changes. In early 2019, you will be able to make changes using a self-service management console.
+
+### How do I add nodes?
+
+To add nodes, you can reach out to [Support]([Support](https://support.cockroachlabs.com/hc/en-us) to request to add nodes. Our team will work with you to update your cluster configurations. We expect this to be self-service next year.
+
+### Do you auto-scale?
+
+Today, we do not automatically scale nodes based on your capacity usage. There are plans to allow auto-scaling in the future.
+
+### Who is responsible for backup?
+
+Cockroach Labs will run daily full backups and hourly incremental backups for all of your managed clusters.
+
+## Product features
+
+### Are enterprise features like partitioning or change data capture available to me?
+
+Yes, the managed clusters run the enterprise version of CockroachDB and all enterprise features are available to you. We encourage you to work with our Sales Engineering team to set up [partitioning](partitioning.html), [change data capture](change-data-capture.html) and other advanced features, as we have best practices and reference architectures we would be happy to share with you.
+
+## Cluster troubleshooting
+
+### What do I do if my queries are too slow?
+
+To optimize schema design to achieve your performance goals, we recommend working with our Sales Engineering team before you set up your cluster. You can also reach out to [Support](https://support.cockroachlabs.com/hc/en-us).
+
+### Can I use my startup cloud credits?
+
+Today, we do not support cloud credits you may have in your account, as we run the clusters in our own accounts.
+
+### Can you run my CockroachDB cluster on-premise?
+
+Today, we do not have an offering that manages running CockroachDB on customer premises. In the future, we expect to have a private cloud offering that will allow you to run multiple CockroachDB clusters on your premises using the same tools that we use internally for our managed offering.
+
+### Do you have a private cloud offering?
+
+[See above.](#can-you-run-my-cockroachdb-cluster-on-premise)
+
+### Can I see a demo?
+
+Yes, [contact us](https://support.cockroachlabs.com/hc/en-us) and we’d be happy to show you a demo of our managed CockroachDB offering.
+
+### Do you have a UI? How can I see details?
+
+All customers of our managed service have access to the [CockroachDB Admin UI](managed-use-the-admin-ui.html). In early 2019, there will be a self-service management console where you can connect to your cluster, view details of your cluster, and make changes to your existing cluster.
```

Comment:
We might want to make a distinction here: All customers can **monitor and troubleshoot** their clusters by accessing the Admin UI. In early 2019, they can also **manage** their clusters using the self-service management console.